lib/configs.js

```diff
@@ -1,25 +1,27 @@
+'use strict'
+
 // Internal Modules
-const pkg = require('../package.json');
+const pkg = require('../package.json')
 
 module.exports = {
-    LOG_LEVELS: ['TRACE', 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL']
-    , DEFAULT_REQUEST_HEADER: { 'Content-Type': 'application/json; charset=UTF-8' }
-    , DEFAULT_REQUEST_TIMEOUT: 180000
-    , DEFAULT_USER_AGENT: `${pkg.name}/${pkg.version}`
-    , MS_IN_A_DAY: 86400000
-    , MAX_REQUEST_TIMEOUT: 300000
-    , MAX_LINE_LENGTH: 32000
-    , MAX_INPUT_LENGTH: 80
-    , FLUSH_INTERVAL: 250
-    , FLUSH_BYTE_LIMIT: (process.env.test === 'test') ? 320000 : 5000000
```

Comment:
we no longer want to be able to configure the FLUSH_INTERVAL or variables below based off of process.env.test?

Reply:
Having any kind of "test" code in the main code base is not necessary.  We can control these things by overriding the constructor instead.  That approach accepts an optional override and defaults to the constant.  This is preferred over  a conditional in a constants file (cause then it's not constant!) and always using the constant.

I added all of these things to the `createOptions` function which can then set them in the instance.